docs/usage.rst

```diff
@@ -6,86 +6,82 @@ To use TIA Toolbox in a project::
 
     import tiatoolbox
 
-
-----------
-Dataloader
-----------
-.. automodule:: tiatoolbox.dataloader
-
-^^^^^^^^^^^^^^^^^^^^
-dataloader.wsireader
-^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^
+Reading WSI Image Data
+^^^^^^^^^^^^^^^^^^^^^^
 
 .. currentmodule:: tiatoolbox.dataloader.wsireader
 .. autoclass:: WSIReader
-    :members: read_region, slide_thumbnail, save_tiles
+    :members: read_rect, read_bounds, read_region, slide_thumbnail, save_tiles
 
 .. autoclass:: OpenSlideWSIReader
-    :members: slide_info, read_region, slide_thumbnail, save_tiles
+    :members: slide_info
```

Comment:
Do we plan to change `slide_info` to `info` in this PR or separate?

Reply:
I'm happy either way. Can do it now if you like.

Reply:
Done in https://github.com/TIA-Lab/tiatoolbox/pull/40/commits/b052a1d26da8c6dff978066bf37d660382164ee5